roles/scaffold_rm_facts/templates/module_utils/network_os/config/resource/resource.py.j2

```diff
@@ -13,6 +13,14 @@ created
 from ansible.module_utils.network.common.cfg.base import ConfigBase
 from ansible.module_utils.network.common.utils import to_list
 from {{ import_path }}.{{ network_os }}.facts.facts import Facts
+{% if transport.netconf %}
+from ansible.module_utils.network.junos.junos import (locked_config,
+                                                      load_config,
+                                                      commit_configuration,
+                                                      discard_changes,
+                                                      tostring)
+from ansible.module_utils.network.junos.utils import utils as junosutils
```

Comment:
This is changed to `from ansible.module_utils.network.common.netconf import build_root_xml_node, build_child_xml_node` as part of PR `https://github.com/ansible/ansible/pull/58871`